src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/FancyZonesViewModel.cs

```diff
@@ -351,7 +362,7 @@ public bool MakeDraggedWindowsTransparent
             }
         }
 
-        public Color ZoneHighlightColor
+        public string ZoneHighlightColor
```

Comment:
Why do we need to convert Color to string?  I personally prefer the the ability of a class to enforce class invariants rather than a generic string which forces clients of the object to make sure the values are valid.

Reply:
Color is from Windows.UI.